lib/membrane_mp4/container.ex

```diff
@@ -26,38 +26,38 @@ defmodule Membrane.MP4.Container do
   @doc """
   Parses binary data to MP4 according to `#{inspect(Schema)}.schema/0`.
   """
-  @spec parse(binary) :: {:ok, t} | {:error, parse_error_context_t}
+  @spec parse(binary) :: {:ok, t, binary()} | {:error, parse_error_context_t}
```

Comment:
```suggestion
  @spec parse(binary) :: {:ok, t, leftover :: binary()} | {:error, parse_error_context_t}
```

I'd add some explanation what the returned binary is either by using a named type or in `@doc`. Same for `parse/2`